blocks/v2-dealer-locator-teaser/v2-dealer-locator-teaser.css

```diff
@@ -16,42 +16,42 @@
 .v2-dealer-locator-teaser .v2-dlt__content-wrapper {
   width: 100%;
   max-width: var(--wrapper-width);
+  padding: var(--section-div-padding);
 }
 
 .v2-dealer-locator-teaser .v2-dlt__content {
   background-color: var(--c-white);
   text-align: center;
-  padding: 32px 24px 24px;
-  margin: 0 24px 24px;
+  padding: 24px;
   border-radius: 8px;
```

Comment:
I see that` border radius: 12px`, can you change that 
https://www.figma.com/file/IcqreRKnNYyDVMc28yoOeh/Homepage?node-id=4259%3A4868&mode=dev